packages/medusa/src/models/product-category-product.ts

```diff
@@ -0,0 +1,71 @@
+import { generateEntityId } from "../utils/generate-entity-id"
+import { BaseEntity } from "../interfaces/models/base-entity"
+import { Product, ProductCategory } from "."
+import {
+  BeforeInsert,
+  Index,
+  Entity,
+  Column,
+  JoinColumn,
+  ManyToOne,
+} from "typeorm"
+
+@Entity()
+export class ProductCategoryProduct extends BaseEntity {
```

Comment:
todo: followup on @pKorsholm comments, we usually don't create an entity or repo for pivot table

Reply:
Thanks for mentioning, didn't know that. I've removed the model and repo. 

Reason I added it was for the consumer of a medusa service who wants to reference the model or repo, like all other entities we provide. But I also understand that they can plug that in themselves. There is no usecase for the core to need them.

Reply:
cc @pKorsholm

Reply:
In general nobody reference them. When loading the relation typeorm manage itself the pivot table and return the target entity

Reply:
understood, does this mean that the migration also should be dropped?

Reply:
No the migration should still exists 👍